integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiServiceMigration.java

```diff
@@ -351,4 +399,40 @@ private boolean checkJmsServerRuntime(String managedServer) {
     return true;
   }
 
+  /*
+   * Verify the JTA Recovery Service Runtime through rest API.
+   * Get the JTA Recovery Service Runtime for a server on a 
+   * specified managed server.
+   * @param managedServer name of the server to look for RecoveyServerRuntime
+   * @param recoveryService name of RecoveyServerRuntime (managed server) 
+   * @param active is the recovery active (true or false )
+   * @returns true if MBEAN is found otherwise false
+   **/
+  private boolean checkJtaRecoveryServiceRuntime(String managedServer, String recoveryService, String active) {
+    ExecResult result = null;
+    int adminServiceNodePort
+        = getServiceNodePort(domainNamespace, getExternalServicePodName(adminServerPodName), "default");
+    StringBuffer curlString = new StringBuffer("curl --user weblogic:welcome1 ");
+    curlString.append("\"http://" + K8S_NODEPORT_HOST + ":" + adminServiceNodePort)
```

Comment:
hard coded values - switch to curl --user " + ADMIN_USERNAME_DEFAULT + ":" + ADMIN_PASSWORD_DEFAULT

Reply:
done